### ISSUE TITLE
Autoreject RuntimeWarning and figure display problem

### ISSUE THREAD

**Issue by ibr2011** [autoreject/autoreject#242]
Greetings, 

I have encountered two issues while using Autoreject. The first issue was a RuntimeWarning:
`\site-packages\mne\bem.py:974: RuntimeWarning: Mean of empty slice.
  radius_init = radii.mean()`
`site-packages\numpy\core\_methods.py:188: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)`
`\site-packages\numpy\core\fromnumeric.py:3419: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,`
`site-packages\numpy\core\_methods.py:180: RuntimeWarning: invalid value encountered in true_divide
  ret = um.true_divide(`

I got this after I ran the following code: 

````
n_interpolates = np.array([1, 4, 32])
consensus_percs = np.linspace(0, 1.0, 11)

ar = AutoReject(n_interpolates, consensus_percs, cv=10, picks='eeg', random_state=42, verbose='tqdm')

Pilot_5_testing_autoreject_epo, Pilot_5_testing_autoreject_log = ar.fit_transform(Pilot_5_testing, return_log=True)
````

the second issue was when I tried to plot a figure. 

![Figure_1](https://user-images.githubusercontent.com/17813340/138901626-0d600bd6-8b14-40e8-b185-bb1b4d17ccd2.png)

**Comment by ibr2011** [comment]
Greetings @jasmainak ,

I hope you are doing well. 

The warning message occurred again using this dataset. 
Link: https://github.com/ibr2011/Behavior_analysis.git

Could you please check it?

Regards

**Comment by jasmainak** [comment]
@ibr2011 thanks for the report. I'll try to look sometime early next week!

**Comment by jasmainak** [comment]
@ibr2011 I don't get the warning message with your dataset. Can you share what you get with:

```py
import mne
mne.sys_info()
```

here's mine and this gives no warning:

```py
Platform:       macOS-10.16-x86_64-i386-64bit
Python:         3.9.4 (default, Apr  9 2021, 09:32:38)  [Clang 10.0.0 ]
Executable:     /Users/mainak/anaconda3/envs/autoreject/bin/python
CPU:            i386: 4 cores
Memory:         8.0 GB

mne:            0.24.0
numpy:          1.20.1 {blas=mkl_rt, lapack=mkl_rt}
scipy:          1.6.2
matplotlib:     3.3.4 {backend=MacOSX}

sklearn:        0.24.1
numba:          0.53.1
nibabel:        Not found
nilearn:        Not found
dipy:           Not found
cupy:           Not found
pandas:         1.2.4
mayavi:         Not found
pyvista:        Not found
pyvistaqt:      Not found
ipyvtklink:     Not found
vtk:            Not found
PyQt5:          5.9.2
ipympl:         Not found
mne_qt_browser: Not found
```

and here's my code:

```py
from mne import read_epochs
from autoreject import AutoReject

ar = AutoReject()
epochs = read_epochs()
epochs_clean = ar.fit_transform(epochs)
```

can you make sure you are running latest version of both autoreject and MNE?

**Comment by ibr2011** [comment]
Hello @jasmainak ,

Thank you for your reply.

Attached is the output of `mne.sys_info()`

![sys_info](https://user-images.githubusercontent.com/17813340/141305231-a33ad09a-855c-419d-9bba-de9a2df33d75.png)

**Comment by jasmainak** [comment]
also just to confirm, what is the `autoreject` version you are using?

**Comment by ibr2011** [comment]
Hello @jasmainak ,

Thank you for your patience. 

I think it is '0.3.dev0'

**Comment by jasmainak** [comment]
nopes I can't reproduce :(

![image](https://user-images.githubusercontent.com/15852194/141370916-6a91c1fa-73c5-4f0f-b2d2-052b6a16a698.png)

are you sure you [gave me the right dataset](https://github.com/autoreject/autoreject/issues/242#issuecomment-955872578)? I can't help unless I can reproduce this warning on another computer ...

**Comment by ibr2011** [comment]
This is bizarre. It was the right one. Link Below contains another dataset which the warning message appeared when I tried to perform autoreject.
Link: [](https://drive.google.com/file/d/1V5lyhDdBbRalExpbvYkwj5E4I_9ylwQJ/view?usp=sharing)

I truly apologize for taking lots of your time. If you couldn't reproduce the warning message, I'll attend the office hours of MNE on Discord on Friday if you plan to be there and share my screen. 

Kind regards

**Comment by jasmainak** [comment]
Do you want to send me an email on mainakjas [at] gmail [dot] com 

Let's set up a time to talk on zoom. I'm not sure to be there on Discord but we can try to debug this separately
